packages/platform/src/HttpApiBuilder.ts

```diff
@@ -971,7 +971,7 @@ export const securityDecode = <Security extends HttpApiSecurity.HttpApiSecurity>
         password: Redacted.make("")
       } as any
       return HttpServerRequest.HttpServerRequest.pipe(
-        Effect.flatMap((request) => Encoding.decodeBase64String(request.headers.authorization ?? "")),
+        Effect.flatMap((request) => Encoding.decodeBase64String(request.headers.authorization?.split(" ")[1] ?? "")),
```

Comment:
Nitpicky, but I'd probably use the same approach as for the Bearer token above:

```ts
(request.headers.authorization ?? "").slice(basicLen)
```

Reply:
@fubhy I'm AFK at the moment and have no preference over stylistic changes so please feel free to amend this PR as you see best